reader-osm/src/main/java/com/graphhopper/reader/osm/OSMReader.java

```diff
@@ -698,11 +698,11 @@ EdgeIteratorState addEdge(int fromIndex, int toIndex, PointList pointList, IntsR
                     pillarNodes.add(lat, lon);
             }
         }
-        if (towerNodeDistance < 0.0001) {
+        if (towerNodeDistance < 0.001) {
             // As investigation shows often two paths should have crossed via one identical point 
             // but end up in two very close points.
             zeroCounter++;
-            towerNodeDistance = 0.0001;
+            towerNodeDistance = 0.001;
```

Comment:
... and avoid 0 distance

Reply:
Would it make sense to define a constant for the minimum storeable distance? These kind of magic numbers always come back to hunt you.

Reply:
Yeah, I don't like it too :) . Still "providing an API" for this limitation is uglier IMO at least when this is only needed in one place. Will add a comment to the BaseGraph to remind us of this here when refactoring.

Reply:
An API would be nice but i think that a simple `MIN_STOREABLE_DISTANCE` constant with a comment would be sufficient for readability.